includes/class-wc-siftscience-admin.php

```diff
@@ -163,9 +163,26 @@ private function output_settings_stats() {
 				return;
 			}
 
-			$stats = json_decode( $stats );
+			$stats = json_decode( $stats , true );
+			$stats_array = $stats;
 			$stats = json_encode( $stats, JSON_PRETTY_PRINT );
-			echo '<pre>' . $stats . '</pre>';
+
+			$stats_keys = array_keys( $stats_array );
+						
+			for ( $i = 0 ; $i < count( $stats_keys ) ; $i++ ) { 
+				$key = $stats_keys[ $i ];
+				$present_key = str_replace( '_', ' ', $key );
```

Comment:
this string is actually the name of the function. So I don't think it makes sense to make it look like "English".

Also, I was thinking of changing the format of the string to look like: `[class name]::[function name]`